### PR TITLE
Add new study: Wilson_2004

### PR DESCRIPTION
**Pull request by ehwenk**
* transcribed from manuscript

### CODE REVIEW

File: data/Wilson_2004/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  observation_number: 2
+  measurement_remarks: data from Vegetation Species List and Response Database (DEC_2008)
+substitutions:
+- trait_name: resprouting_capacity
+  find: Seeders
+  replace: fire_killed
+- trait_name: resprouting_capacity
+  find: Resprouters
+  replace: resprouts
+- trait_name: resprouting_capacity
```
Comment (yangsophieee) [inline data/Wilson_2004/metadata.yml:182]: Duplicated?

**Review by yangsophieee — APPROVED** [review]

File: data/Wilson_2004/data.csv
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Wilson_2004/data.csv:1]: I've added Banksia elegans, but the other two species were already in the main table with identical numbers (when rounded to the closest year).

File: data/Wilson_2004/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    1998-2009), from recovery plans for threatened species and Threatened Ecological
+    Communities of the area that are covered by State and Commonwealth legislation
+    and require specific fire regimes for protection (Brown et al., 1998; Evans et
+    al., 2003). As species responses vary from site to site (Vivian et al., 2010)
+    and as most of the plant post-fire responses were obtained from the northern Jarrah
+    forest, we supplemented this data by recording the post-fire responses of a number
+    of flora species during post-fire and chronosequence floristic surveys conducted
+    in the Banksia woodlands (Mickle et al., 2010a, 2010b, 2009; Valentine et al.,
+    2009a; Wilson et al., 2010a).
+- var_in: juvenile_period_years_Mickle_2009
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Wilson_2004/metadata.yml:113]: done

[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Wilson_2004/metadata.yml:113]: (I added it as a context)

File: data/Wilson_2004/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    description: Banksia overstorey and sporadic stands of Eucalyptus, Corymbia and
+      Allocasuarina, and an understorey consisting mainly of low shrubs from the Myrtaceae,
+      Fabaceae and Proteaceae families (Barrett and Pin Tay, 2005; Gibson et al.,
+      1994), and a variety of floristic communities including coastal shrublands on
+      shallow sands and Acacia shurblands.
+    notes: A distinct water catchment of approximately 220,000 ha and is located on
+      the Swan Coastal Plain just north of Perth.
+contexts:
+- context_property: replicate observations
+  category: temporal
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Wilson_2004/metadata.yml:89]: I've reworked the entire dataset, mapping these as separate rows with different source_id's

**Pull request merged**